src/main/sf_496C0.c

```diff
@@ -1,71 +1,2995 @@
-#include "common.h"
+#include "global.h"
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/main/sf_496C0/func_80048AC0.s")
+#include "variables.h"
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/main/sf_496C0/func_80048CC4.s")
+extern Gfx D_1024AC0[];
+extern Gfx D_60320E0[];
+extern Gfx* D_7010970[];
+extern Gfx* D_E000000;
+extern Gfx* D_E003AB0;
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/main/sf_496C0/func_80048E40.s")
+extern s32 D_800C9E90[];
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/main/sf_496C0/func_80049630.s")
+extern AnimationHeader D_60265B4[];
+extern AnimationHeader D_602B8DC[];
+extern AnimationHeader D_60034C4[];
+extern AnimationHeader D_602A2CC[];
+extern AnimationHeader D_602CEB4[];
+extern AnimationHeader D_602B778[];
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/main/sf_496C0/func_8004990C.s")
+extern f32 D_80178454;
+extern f32 D_80178458;
+extern f32 D_80178464;
+extern f32 D_80178468;
+extern f32 D_80178474;
+extern f32 D_80178478;
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/main/sf_496C0/func_80049968.s")
+extern Gfx D_400AAE0[];
+extern Gfx D_6000D80[];
+extern Gfx D_600BAA0[];
+extern Gfx D_60132A0[];
+extern Gfx D_60132B0[];
+extern Gfx D_6014A40[];
+extern Gfx D_6020810[];
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/main/sf_496C0/func_80049A9C.s")
+extern AnimationHeader D_6020A40[];
+extern AnimationHeader D_6020C6C[];
+extern AnimationHeader D_602D140[];
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/main/sf_496C0/func_80049B44.s")
+Vec3f D_800C9F60[] = { {
+                           2000.0f,
+                           2000.0f,
+                           2000.0f,
+                       },
+                       {
+                           -2000.0f,
+                           2000.0f,
+                           2000.0f,
+                       },
+                       {
+                           0.0f,
+                           -3000.0f,
+                           3000.0f,
+                       },
+                       {
+                           5000.0f,
+                           -3000.0f,
+                           -3000.0f,
+                       } };
```

Comment:
trailing comma might make it look better, same with the other ones
```suggestion
}, };